src/components/BrokerView/index.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+import {
+  IServiceBinding,
+  IServiceBroker,
+  IServiceClass,
+  IServiceInstance,
+  IServicePlan,
+} from "../../shared/ServiceCatalog";
+import { Card, CardContainer } from "../Card";
+import SyncButton from "../SyncButton";
+
+interface IBrokerViewProps {
+  bindings: IServiceBinding[];
+  broker: IServiceBroker | null;
+  classes: IServiceClass[];
+  getCatalog: () => Promise<any>;
+  instances: IServiceInstance[];
+  plans: IServicePlan[];
+  sync: (broker: IServiceBroker) => Promise<any>;
+}
+
+export class BrokerView extends React.PureComponent<IBrokerViewProps> {
+  public async componentDidMount() {
+    this.props.getCatalog();
+  }
+
+  public render() {
+    const { bindings, broker, instances } = this.props;
+
+    return (
+      <div className="broker">
+        {broker && (
+          <div>
+            <h1>{broker.metadata.name}</h1>
+            <div>Catalog last updated at {broker.status.lastCatalogRetrievalTime}</div>
+            <div
+              style={{
+                display: "flex",
+                flexDirection: "row",
+              }}
+            >
+              <Link to={window.location.pathname + "/classes"}>
+                <button className="button button-primary">Provision New Service</button>
+              </Link>
+              <SyncButton sync={this.props.sync} broker={broker} />
+            </div>
+            <h3>Service Instances</h3>
```

Comment:
I'm wondering if the list of instances and binding should be in the outer view in `/services` rather than per-broker. I think it would be useful to see all the services/bindings from all brokers in the outer view.